File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/invest-vetting-batch_item.ts


import { IInvestVettingBatch_Item } from './iinvest-vetting-batch_item';

export class InvestVettingBatch_Item implements IInvestVettingBatch_Item {
  
	public VettingBatchID: number = 0;
	public PersonsVettingID: number = 0;
	public PersonID: number = 0;
	public Name1: string = "";
	public Name2: string = "";
	public Name3: string = "";
	public Name4: string = "";
	public Name5: string = "";
	public FirstMiddleNames: string = "";
	public LastNames: string = "";
	public FatherName: string = "";
	public MotherName: string = "";
	public DOB?: Date;
	public Gender: string = "";
	public NationalID: string = "";
	public POBCityName: string = "";
	public POBStateName: string = "";
	public POBCountryName: string = "";
	public UnitName: string = "";
	public UnitType: string = "";
	public JobTitle: string = "";
	public RankName: string = "";
	public ModifiedDate: Date = new Date(0);
	public UnitAliasJson: string = "";
  
}

File: GTTS/INL.VettingService.Models/InvestVettingBatch_Item.cs
using System;

namespace INL.VettingService.Models
{
    public class InvestVettingBatch_Item : IInvestVettingBatch_Item
    {
        public long VettingBatchID { get; set; }
        public long PersonsVettingID { get; set; }
        public long PersonID { get; set; }
        public string Name1 { get; set; }
        public string Name2 { get; set; }
        public string Name3 { get; set; }
        public string Name4 { get; set; }
        public string Name5 { get; set; }
        public string FirstMiddleNames { get; set; }
        public string LastNames { get; set; }
        public string FatherName { get; set; }
        public string MotherName { get; set; }
        public DateTime? DOB { get; set; }
        public char Gender { get; set; }
        public string NationalID { get; set; }
        public string POBCityName { get; set; }
        public string POBStateName { get; set; }
        public string POBCountryName { get; set; }
        public string UnitName { get; set; }
        public string UnitType { get; set; }
        public string JobTitle { get; set; }
        public string RankName { get; set; }
        public DateTime ModifiedDate { get; set; }
        public string UnitAliasJson { get; set; }
    }
}
